### PR TITLE
address golint warnings for sub-packages of control/plugin/*

### PR DESCRIPTION
**Pull request by candysmurf**

### CODE REVIEW

File: control/plugin/cpolicy/integer.go [hunk 1]
```diff
@@ -59,6 +58,7 @@ func NewIntegerRule(key string, req bool, opts ...int) (*IntRule, error) {
 	}, nil
 }
 
+// Type returns string representation of integer
```
Comment (danielscottt) [inline control/plugin/cpolicy/integer.go]: Type is the _name_ of the rule type

File: control/plugin/cpolicy/node.go [hunk 2]
```diff
@@ -31,11 +31,13 @@ import (
 	"github.com/intelsdi-x/snap/pkg/ctree"
 )
 
+// ProcessingErrors struct type contains array of errors.
```
Comment (danielscottt) [inline control/plugin/cpolicy/node.go]: Processing errors contains the errors encountered when processing the policy

File: control/plugin/cpolicy/node.go [hunk 3]
```diff
@@ -220,18 +236,18 @@ func addRulesToConfigPolicyNode(rules map[string]interface{}, cpn *ConfigPolicyN
 				r, _ := NewIntegerRule(k, req)
 				if d, ok := rule["default"]; ok {
 					// json encoding an int results in a float when decoding
-					def_, _ := d.(float64)
-					def := int(def_)
+					def0, _ := d.(float64)
```
Comment (danielscottt) [inline control/plugin/cpolicy/node.go]: I like `def_` more than `def0`

File: control/plugin/cpolicy/string.go [hunk 4]
```diff
@@ -135,15 +136,17 @@ func (s *StringRule) Default() ctypes.ConfigValue {
 	return nil
 }
 
-// Indicates this rule is required.
+// Required indicates this rule is required.
 func (s *StringRule) Required() bool {
 	return s.required
 }
 
+// Minimum returns nil
```
Comment (danielscottt) [inline control/plugin/cpolicy/string.go]: string rules do not have mins or maxes

File: control/plugin/client/cache.go [hunk 5]
```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/intelsdi-x/snap/pkg/chrono"
 )
 
-// the time limit for which a cache entry is valid.
+// GlobalCacheExpiration the time limit for which a cache entry is valid.
```
Comment (tiffanyfj) [inline control/plugin/client/cache.go]: GlobalCacheExpiration **is** the time limit for which a cache entry is valid.

**Comment by candysmurf** [comment]
Closing for higher priority items. We'll get back to it later.

**Pull request closed without merging**